services/player-service/application/src/test/java/com/kjeldsen/player/application/usecases/CreatePlayerUseCaseShould.java

```diff
@@ -0,0 +1,46 @@
+package com.kjeldsen.player.application.usecases;
+
+import com.kjeldsen.player.application.repositories.PlayerPositionTendencyReadRepository;
+import com.kjeldsen.player.application.repositories.PlayerWriteRepository;
+import com.kjeldsen.player.domain.Player;
+import com.kjeldsen.player.domain.PlayerAge;
+import com.kjeldsen.player.domain.PlayerPosition;
+import com.kjeldsen.player.domain.PlayerPositionTendency;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CreatePlayerUseCaseShould {
```

Comment:
test classes should end either with Test (surefire reports) or TestIT (failsafe reports)

Reply:
so we. can use jacoco to propagate coverage to Sonar reports

Reply:
Done! 👍